examples/workshop/contract-example-reply/src/lib.rs

```diff
@@ -0,0 +1,77 @@
+#![no_std]
+#![feature(const_btree_new)]
+
+extern crate alloc;
```

Comment:
We don't use btrees here. And most common alloc features are provided by `prelude` mod
```suggestion
```